pkg/security/probe/activity_dump.go

```diff
@@ -1392,21 +1411,37 @@ type BindNode struct {
 
 // SocketNode is used to store a Socket node and associated events
 type SocketNode struct {
-	Family string   `msg:"family"`
-	Bind   BindNode `msg:"bind"`
+	Family string      `msg:"family"`
+	Bind   []*BindNode `msg:"bind,omitempty"`
```

Comment:
Why do we use an array here and not a map like in other insertion-friendly fields (like files) ? Other question: why `*BindNode` and not `BindNode` ? It's just a string and an integer, it's not too big I think

Reply:
Yeah I didn't use a map because at some point we'll be inserting the protocol too and I didn't if it would be better to insert by family or protocol ... As for the *BindNode, no particular reason: I was simply following the nesting patterns of the other structures.

Reply:
Alright, not sure this matters a lot for now anyway